### PR TITLE
Verify docs and formatting on Travis CI

### PR DESCRIPTION
**Pull request by prattmic**
This PR consists of several commits.

First I extended ``tools/format.sh`` with more options so that Travis could check all files.

Then I updated and froze ``yapf``, re-running the latest version on all of the files. Since it is a bit finicky from version to version, I have frozen it at the current latest version.

Finally, I added a script to ensure the docs build and enabled both in the Travis config.

### CODE REVIEW

File: client/interop/types.py [hunk 1]
```diff
@@ -94,8 +94,8 @@ class ServerInfo(Serializable):
     """
 
     def __init__(self, message, message_timestamp, server_time):
-        super(ServerInfo, self).__init__(
-            ["message", "message_timestamp", "server_time"])
+        super(ServerInfo,
+              self).__init__(["message", "message_timestamp", "server_time"])
```
Comment (pmtischler) [inline client/interop/types.py]: Is this what we want it to look like?

File: server/auvsi_suas/views/targets_test.py [hunk 2]
```diff
@@ -625,8 +633,8 @@ def test_get_no_image(self):
 
     def test_delete_no_image(self):
         """404 when DELETE image before upload."""
-        response = self.client.delete(
-            targets_id_image_url(args=[self.target_id]))
+        response = self.client.delete(targets_id_image_url(args=[self.target_id
+                                                                 ]))
```
Comment (pmtischler) [inline server/auvsi_suas/views/targets_test.py]: Same here.

**Comment by pmtischler** [comment]
LGTM

File: client/interop/types.py (hunk 1 above)
Comment (prattmic) [inline client/interop/types.py]: No, but I couldn't convince yapf to do anything else. I fixed it and disabled formatting.

File: server/auvsi_suas/views/targets_test.py (hunk 2 above)
Comment (prattmic) [inline server/auvsi_suas/views/targets_test.py]: Same

**Pull request merged**